apps/web/modules/users/views/users-type-public-view.getServerSideProps.tsx

```diff
@@ -234,7 +243,8 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
     slug,
     trpcState: ssr.dehydrate(),
     isBrandingHidden: user?.hideBranding,
-    isSEOIndexable: user?.allowSEOIndexing,
+    isSEOIndexable:
+      user?.profile?.organization?.organizationSettings?.allowSEOIndexing ?? user?.allowSEOIndexing,
```

Comment:
So, if either of them is true, you allow SEO indexing for both personal and organizational profiles. Is this what we need? Isn't that different?

Reply:
yeah right this is not required, as org/team slug will take care of org settings, my bad i missed to remove this after updating team slug.